.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,22 @@
+name: "Lint PR"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
I think it's incorrect to have this triggered on both `pull_request` and `pull_request_target`; judging from [the action's README](https://github.com/amannn/action-semantic-pull-request/tree/v4#event-triggers), it seems like we should be using only `pull_request_target`?
```suggestion
```